etc/picongpu/taurus-tud/V100.tpl

```diff
@@ -31,6 +31,8 @@
 #SBATCH --cpus-per-task=!TBG_coresPerGPU
 #SBATCH --mem=0
 #SBATCH --gres=gpu:!TBG_gpusPerNode
+#SBATCH --exclusive
```

Comment:
I tried that half a year ago and made a bad experience. The scheduler was not able to distribute tasks to nodes correctly.

From my own tests it looks like `--cpus-per-task` produces the wrong task distribution. Adding the `--ntasks-per-node=6`, removing `--cpus-per-task`, and setting `mem=254000` option to the submission script results in correct distribution and should furthermore ensure we allocate a node exclusively. 

But I admit that there are actually two cores (which can be assigned 8 threads) are left over in the node. However, it is not possible to ask for a job with zero memory. So we should indeed use the node exclusively.

I submitted a job using 16 gpus out of 18 available with a total of three nodes in order to see whether my new configuration still works.

Reply:
@steindev Did `--exclusive` work for you? I do not get a warning on submission at least - but the cluster is too full - I can not test it.

Reply:
I tested exclusive. IMO it is also better to use this flag because there are only a few MB (256000MiB) main memory available and when we share the node with other it can result in crashes.

Reply:
Don't you want to add `--ntasks-per-node=6`?